smac/main/smbo.py

```diff
@@ -269,6 +280,17 @@ def optimize(self) -> Configuration | list[Configuration]:
         for callback in self._callbacks:
             callback.on_start(self)
 
+        dask_data_to_scatter = {}
+        if isinstance(self._runner, DaskParallelRunner) and dask_data_to_scatter is not None:
+            dask_data_to_scatter = dict(
+                data_to_scatter=self._runner._client.scatter(data_to_scatter, broadcast=True)
+            )
+        elif dask_data_to_scatter is not None:
+            raise ValueError(
+                "dask_data_to_scatter is valid only for DaskParallelRunner, "
+                f"but {dask_data_to_scatter} was provided for {self._runner.__class__.__name__}"
+            )
+
```

Comment:
The error you're getting from the checks are from here, this suggestion should fix it.

```python
elif any(dask_data_to_scatter):
    ...
```